data/json/npcs/hells_raiders/TALK_BANDIT_LEADER.json

```diff
@@ -0,0 +1,126 @@
+    "responses": [
+      { "text": "Hmm.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2" },
+      {
+        "text": "How'd you get to be called the \"Hell's Raiders\"?",
+        "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_NAME"
+      },
+      { "text": "What do you do these days?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_JOB" },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_NAME",
+    "type": "talk_topic",
+    "dynamic_line": "Surely you've heard of the old biker gang, the Hell's Angels?  A few of our head honchos used to run with 'em, so they named us based off that.  I know a guy who could hook you up with a bike, too.  For a price.",
+    "responses": [
+      { "text": "Hmm.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2" },
+      { "text": "What do you do these days?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_JOB" },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_JOB",
+    "type": "talk_topic",
+    "dynamic_line": "I'm the head honcho of this joint.  If anyone's got a problem, they answer to me.  Most days, I just make sure everything runs smoothly and that nobody dies.",
+    "responses": [
+      { "text": "Hmm.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2" },
+      { "text": "Who exactly are you guys anyway?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_WHO" },
+      { "text": "How'd you come to lead this organization?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_LEAD" },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_LEAD",
+    "type": "talk_topic",
+    "dynamic_line": "I'm not the top of the organization, but I topped this joint when the old guy got smashed through a wall by one of those <swear> <zombie>s, had arms as big as trashcans.  He'd wrote me down in his will as the inheritor of the position, so I took it.",
+    "responses": [
+      { "text": "Sorry to hear about that.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2" },
+      { "text": "Who's above you?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_ABOVE" },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_ABOVE",
+    "type": "talk_topic",
+    "dynamic_line": "I answer to the underboss, got somethin' like 20 capos under him; all of who run operations in a local district.  The underbosses go to the boss, who oversees all the territory and wealth in their region.  That boss is part of a delegate board who run our whole thing, the whole show.  The capos manage the soldiers who do most of the footwork, but I go out a bit too.  We then manage any slaves we got in our possession, sometimes sell 'em to other gangs.  They do all the menial labor like cleaning, farming, etc.",
```

Comment:
Not a gang. Also, the whole selling slaves is more a post-apoc bandit trope than

>   All factions were part of the same overall culture only a few months ago. We’re not at a Mad Max: Fury Road stage yet, and if anyone’s acting that way they’re probably basing it on a major motion picture.

(per the [design doc](https://cataclysmdda.org/lore.html))

Even if there are people who have no problems with this, there are a lot who would/do, so you simply can't present it like that.